styles.css

```diff
@@ -0,0 +1,284 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,400;0,700;1,400;1,700&display=swap');
+
+:root {
+  --primary-text: #000;
+  --secondary-text: #666;
+  --primary-bg: #fff;
+  --secondary-bg: #f7f9fa;
+  --tertiary-bg: #e0f3ff;
+  --warning-color: #ff1f13;
```

Comment:
Inconsistency - no other variable has the word "color" inside it. This one has! 
I would say its better to have all colors prefixed with `color-`. Makes it easy to search/autocomplete a color variable and also when you have more variables later for spaing, sizes etc, color variables would be clear!